libs/execution_context/ebpf_protocol.h

```diff
@@ -54,6 +54,7 @@ typedef enum _ebpf_operation_id
     EBPF_OPERATION_RING_BUFFER_MAP_MAP_BUFFER,
     EBPF_OPERATION_RING_BUFFER_MAP_UNMAP_BUFFER,
     EBPF_OPERATION_EPOCH_SYNCHRONIZE,
+    EBPF_OPERATION_LINK_LEGACY_MODE,
```

Comment:
The other values above all have a verb in the name.  What's the verb in this name?